### PR TITLE
(*) Add class on close button component

### PR DESCRIPTION
**Pull request by gdsrosa**

### CODE REVIEW

File: src/TourPortal.js [hunk 1]
```diff
@@ -538,15 +538,17 @@ class TourPortal extends Component {
...
+                        ? nextButton
+                        : null
                     }
                   />
                 )}
               </Controls>
             )}
 
-            {showCloseButton ? <Close onClick={onRequestClose} /> : null}
+            {showCloseButton ? (
```
Comment (nelsonwong1012) [inline src/TourPortal.js]: Would `showCloseButton && <Close onClick={onRequestClose} className="reactour__close" />` work?

**Comment by nelsonwong1012** [comment]
I believe you accidentally pushed your `yarn.lock`.

File: src/TourPortal.js [hunk 2]
```diff
@@ -478,7 +478,7 @@ class TourPortal extends Component {
             {steps[current] &&
               (typeof steps[current].content === 'function'
                 ? steps[current].content({
-                    close: onRequestClose
+                    close: onRequestClose,
```
Comment (nelsonwong1012) [inline src/TourPortal.js]: Good catch! Would be nice to add a linter to this project to catch errors like this.

File: src/TourPortal.js [hunk 3]
```diff
@@ -527,8 +527,8 @@ class TourPortal extends Component {
                           ? onRequestClose
                           : () => {}
                         : typeof nextStep === 'function'
-                          ? nextStep
```
Comment (nelsonwong1012) [inline src/TourPortal.js]: My preference but I actually like the indents.

File: src/TourPortal.js (hunk 2 above)
Comment (elrumordelaluz) [reply src/TourPortal.js]: absolutely agree! mind make a PR?

File: src/TourPortal.js (hunk 3 above)
Comment (elrumordelaluz) [reply src/TourPortal.js]: here is working `.prettierrc`, you mean change settings there?

File: src/TourPortal.js (hunk 1 above)
Comment (elrumordelaluz) [reply src/TourPortal.js]: I think so…, thanks!

**Comment by bogdancss** [comment]
any update on this?

**Comment by PeterKottas** [comment]
Also interested in this. In the meantime, you can do this:
```
.reactour__helper {
  > button:last-of-type {
  }
}
```

**Comment by PeterKottas** [comment]
Lovely!

**Pull request closed without merging**